delta_backend/src/extractor.py

```diff
@@ -48,8 +48,11 @@ def _get_person_names(self):
         person_forename = " ".join(selected_name.get("given", []))
         person_surname = selected_name.get("family", "")
 
-        return person_forename, person_surname
-
+        if person_forename and person_surname:
```

Comment:
What if there were two valid names at the occurrence datetime, the first only had a `given` but the second had a `given` and `family`?

Reply:
Do we need something like this instead?
```
        complete_names = [n for n in names if n.get("given", []) and n.get("family", "")]
        selected_name = self._get_valid_names(complete_names, occurrence_time)
```